seqr/views/apis/users_api_tests.py

```diff
@@ -53,7 +52,6 @@ def test_get_project_collaborator_options(self, mock_analyst_group):
         users.update(self.COLLABORATOR_NAMES)
         self.assertSetEqual(set(response_json.keys()), users)
         self.assertSetEqual(set(response_json[ANALYST_USERNAME].keys()), USER_OPTION_FIELDS)
-        self.assertTrue(response_json[ANALYST_USERNAME]['isAnalyst'])
```

Comment:
You may want to add tests for the Anvil case and test the fields of `username` and `email` because they have different behaviors for Anvil users.

Reply:
there is no anvil field in this data though?? `USER_OPTION_FIELDS = {'displayName', 'username', 'email'}`